governance/remuneration/wave_1/nader-dabit-1.md

```diff
@@ -0,0 +1,18 @@
+# Nader Dabit 6 month invoice
+
+## Hours worked
+
+12 hrs: 
+* Grant Reviews (over the course of the past 6 months)
+
+Total hours: 12
+
+## Total Comp
+
+* Rate: $150 per hour
+* Total hours worked: 12
+* Total hourly pay: 1800 USDC
+* Current 7-day average: 1USDC: 1.60RAD
+* Total: 1125 RAD
```

Comment:
@dabit3 thanks for this. 

I've added a python notebook that does all the heavy lifting (finds monthly average based on last 7 days of each month)
https://github.com/radicle-dev/radicle-grants/blob/main/analytics/rgp_compensation_automation.ipynb

### Main takeaway:
* For 12 hours over the last 6 months of prices, the 1,125 RAD you've proposed is too high 
* It would be roughly 467 RAD 

### My proposal:
* The 1:1 advice you provide and help on marketing (Twitter) is well worth the 1,125 RAD you mentioned
* Can you please "Marketing and technical consulting" that covers what you'd like to get paid?

Making this request so that we have a written record of accounting.

> Note: if this is not done in 3 days, I will start a transaction for 1,125 RAD as-is